op-bindings/artifacts.json

```diff
@@ -1,45 +1,152 @@
-[
-  "SystemConfig",
-  "L1CrossDomainMessenger",
-  "L1StandardBridge",
-  "OptimismPortal",
-  "L2OutputOracle",
-  "AddressManager",
-  "L1Block",
-  "L2ToL1MessagePasser",
-  "GasPriceOracle",
-  "L2CrossDomainMessenger",
-  "L2StandardBridge",
-  "L2ERC721Bridge",
-  "L1ERC721Bridge",
-  "OptimismMintableERC721Factory",
-  "SequencerFeeVault",
-  "BaseFeeVault",
-  "L1FeeVault",
-  "OptimismMintableERC20Factory",
-  "OptimismMintableERC20",
-  "LegacyERC20ETH",
-  "Proxy",
-  "ProxyAdmin",
-  "LegacyMessagePasser",
-  "ERC20",
-  "WETH9",
-  "DeployerWhitelist",
-  "L1BlockNumber",
-  "DisputeGameFactory",
-  "FaultDisputeGame",
-  "AlphabetVM",
-  "StandardBridge",
-  "CrossDomainMessenger",
-  "MIPS",
-  "PreimageOracle",
-  "BlockOracle",
-  "EAS",
-  "SchemaRegistry",
-  "ProtocolVersions",
-  "Safe",
-  "SafeProxyFactory",
-  "DelayedVetoable",
-  "ISemver",
-  "StorageSetter"
-]
+{
+    "local": [
+        "SystemConfig",
+        "L1CrossDomainMessenger",
+        "L1StandardBridge",
+        "OptimismPortal",
+        "L2OutputOracle",
+        "AddressManager",
+        "L1Block",
+        "L2ToL1MessagePasser",
+        "GasPriceOracle",
+        "L2CrossDomainMessenger",
+        "L2StandardBridge",
+        "L2ERC721Bridge",
+        "L1ERC721Bridge",
+        "OptimismMintableERC721Factory",
+        "SequencerFeeVault",
+        "BaseFeeVault",
+        "L1FeeVault",
+        "OptimismMintableERC20Factory",
+        "OptimismMintableERC20",
+        "LegacyERC20ETH",
+        "Proxy",
+        "ProxyAdmin",
+        "LegacyMessagePasser",
+        "ERC20",
+        "WETH9",
+        "DeployerWhitelist",
+        "L1BlockNumber",
+        "DisputeGameFactory",
+        "FaultDisputeGame",
+        "AlphabetVM",
+        "StandardBridge",
+        "CrossDomainMessenger",
+        "MIPS",
+        "PreimageOracle",
+        "BlockOracle",
+        "EAS",
+        "SchemaRegistry",
+        "ProtocolVersions",
+        "Safe",
+        "SafeProxyFactory",
+        "DelayedVetoable",
+        "ISemver",
+        "StorageSetter"
+    ],
+    "remote": [
+        {
+            "name": "Multicall3",
+            "verified": true,
+            "hasImmutables": false,
+            "create2ProxyDeployed": false,
+            "deployments": {
+                "1": "0xcA11bde05977b3631167028862bE2a173976CA11",
+                "10": "0xcA11bde05977b3631167028862bE2a173976CA11"
+            }
+        },
+        {
+            "name": "Create2Deployer",
+            "verified": true,
+            "hasImmutables": false,
+            "create2ProxyDeployed": false,
+            "deployments": {
+                "1": "0xF49600926c7109BD66Ab97a2c036bf696e58Dbc2",
+                "10": "0x13b0D85CcB8bf860b6b79AF3029fCA081AE9beF2"
+            },
+            "useDeploymentBytecodeFromChainId": 1,
+            "useInitBytecodeFromChainId": 1
```

Comment:
Is this the only place we use these variables?

Reply:
Yes, but seemed like a useful feature to remain available for other OP chains that may want to add custom predeploys